README.md

```diff
@@ -27,48 +27,57 @@ jobs:
         uses: azohra/shell-linter@latest
 ```
 
-Run static analysis for a single shell script:
+#### Run static analysis for a single shell script:
 ```yml
       - name: Run Shellcheck
         uses: azohra/shell-linter@latest
         with:
           path: "setup.sh"
 ```
 
-Run static analysis for multiple shell scripts **with or without** extension:
+#### Run static analysis for multiple shell scripts **with or without** extension:
 ```yml
       - name: Run Shellcheck
         uses: azohra/shell-linter@latest
         with:
           path: "setup,deploy.sh"
 ```
 
-Run static analysis for all the shell scripts in a folder:
+#### Run static analysis for all the shell scripts in a folder:
 ```yml
       - name: Run Shellcheck
         uses: azohra/shell-linter@latest
         with:
           path: "src"
 ```
 
-Run static analysis using a **wildcard** path:
+#### Run static analysis using a **wildcard** path:
 ```yml
       - name: Run Shellcheck
         uses: azohra/shell-linter@latest
         with:
           path: "src/*.sh"
 ```
+#### Exclude files and folders from the static analysis:
+```yml
+      - name: Run Shellcheck
+        uses: azohra/shell-linter@latest
+        with:
+          excluded_paths: "src/setup.sh,tests/unit_tests" 
+```
+Note that `excluded_paths` only accepts paths relative to your project's root directory. However, **do not** include `./` at the beginning of the paths. 
```

Comment:
Really excited for the excluded paths feature. A little decision making on our end regarding the format of the name of this input. Should we go with `excluded_paths` or `excluded-paths`? I saw some examples of input names on the docs ([here](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action) and [here](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions))